PIL/EpsImagePlugin.py

```diff
@@ -330,10 +330,13 @@ def _find_offset(self, fp):
 
         return (length, offset)
 
-    def load(self, scale=1):
+    def load(self, scale=1, size=None):
         # Load EPS via Ghostscript
         if not self.tile:
             return
+        # Override the size of the image if specified
+        if not size is None:
+            self.size = size
         self.im = Ghostscript(self.tile, self.size, self.fp, scale)
         self.mode = self.im.mode
         self.size = self.im.size
```

Comment:
Thanks for your review!

Which line are you referring to? Line 342 (self.size = self.im.size)? If so, my understanding is that the size value passed to Ghostscript(...) is what is used for the gs command which rasterizes the image. The rasterized image is then assigned to im, and would thus have the size specified by the parameter to load(). Or was it something else that I missed?